File: Perez-Meneses-Juan-Carlos/Hilos/thread11_create.cs
using System;
using System.Threading;

class HelloWorldThreads
{
    private const int NUM_THREADS = 5;

    static void PrintHello(object threadId)
    {
        Console.WriteLine($"Hello World! It's me, thread #{threadId}!");
    }

    static void Main(string[] args)
    {
        for (int t = 0; t < NUM_THREADS; t++)
        {
            Console.WriteLine($"In main: creating thread {t}");
            Thread thread = new Thread(PrintHello);
            thread.Start(t);
        }
    }
}

File: Perez-Meneses-Juan-Carlos/Hilos/thread11_create.java
public class HelloWorldThreads {
    private static final int NUM_THREADS = 5;

    public static void main(String[] args) {
        for (int t = 0; t < NUM_THREADS; t++) {
            final int threadId = t;
            System.out.println("In main: creating thread " + threadId);
            new Thread(() -> {
                System.out.println("Hello World! It's me, thread #" + threadId + "!");
            }).start();
        }
    }
}
